config_settings/bazel/compilation_mode/BUILD.bazel

```diff
@@ -2,6 +2,8 @@ load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
 load("@cgrindel_bazel_starlib//bzlformat:defs.bzl", "bzlformat_pkg")
 load(":compilation_modes.bzl", "compilation_modes")
 
+package(default_visibility = ["//visibility:public"])
+
```

Comment:
Instead of setting this on the package, let's set the `visibility` attribute on the `config_setting`.

Reply:
Done, I had to set `visibility = ["//visibility:public"]` to associated `bzl_library` rules, let me know if it would be better to set a different level of visibility.